.github/workflows/release.yml

```diff
@@ -133,7 +133,7 @@ jobs:
                   install-deps: "true"
 
             - name: Configure CMake
-              run: cmake -DCMAKE_BUILD_TYPE=Release -DANTIMICROX_PKG_VERSION="GitHub Windows Release" -B '${{ github.workspace }}'/build
+              run: cmake -DCMAKE_BUILD_TYPE=Release -DCHECK_FOR_UPDATES=ON -DANTIMICROX_PKG_VERSION="GitHub Windows Release" -B '${{ github.workspace }}'/build
```

Comment:
@gombosg 
> Also, what do you think about disabling this in repository and flatpak versions, where users are unable to update by themselves? 

This is disabled by default. Only Windows build uses this feature

Reply:
Sounds cool then! Window$ users need some handholding 🤡